.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
Do we want Eclipse preferences in the repo? I"d rather not, but I don't expect they'd get in the way.